src/eatery/tests.py

```diff
@@ -1,16 +1,10 @@
 from os import environ
 
-from models import Request, Test, TestGroup
+from models import Pod, Request, Test, TestGroup
 
 BASE_URL = environ["EATERY_BACKEND_URL"]
+URL_PARAMS = "?query=query%7Beateries%20%7Bname%7D%7D"
```

Comment:
if you "unmangle" this, you would actually get something like this: `query=query { eateries { name } }`

however, we probably want to "store" the unmangled version in this codebase, and just mangle it on the spot instead –– something like this:

```python
URL_PARAMS = "query=query { eateries { name } }".mangle() # we're pretending that mangle is the method that converts this
```